src/modules/status.rs

```diff
@@ -0,0 +1,36 @@
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::status::StatusConfig;
+use crate::formatter::StringFormatter;
+
+/// Creates a module with the status of the last command
+///
+/// Will display the status only if it is not 0
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let exit_code_default = std::string::String::from("0");
+    let exit_code = context
+        .properties
+        .get("status_code")
+        .unwrap_or(&exit_code_default);
```

Comment:
Any particular reason why'd you allocate a string here instead of doing something like
```suggestion
    let exit_code = context
        .properties
        .get("status_code")
        .map_or("0", String::as_str);
```

Reply:
No specific reason — I just did it as in `character.rs`